src/Teambuilder/client.cpp

```diff
@@ -2055,7 +2057,11 @@ void Client::seeChallenge(const ChallengeInfo &c)
     int challengeId = freeChallengeId();
     if (playerExist(c))
     {
-        if (!call("beforeChallengeReceived(int,int,QString,int)", challengeId, c.opponent(), c.desttier, static_cast<int>(c.clauses))) {
+        if (mychallenges.size() > 4) {
```

Comment:
Yeah I didn't mean to leave it in seeinfo(), I didn't know which one did what at the time and forgot to take it out when I pushed. I just wanted the exploit where any user and crash a desktop user by spamming them with challenges to be fixed. Something no one thought was an issue needing to be solved.